Documentation/compatibility/winforms-accessibility-changes-472.md

```diff
@@ -0,0 +1,107 @@
+
+__Use of OS-defined colors in High Contrast themes__
+- The drop down arrow arrow of the `T:System.Windows.Forms.ToolStripDropDownButton` now has uses now use OS-defined colors in High Contrast theme.
+- `T:System.Windows.Forms.Button`, `T:System.Windows.Forms.RadioButton` and `T:System.Windows.Forms.CheckBox` controls with `P:System.Windows.Forms.Control.FlatStyle` set to <xref:System.Windows.Forms.FlatStyle.Flat?displayProperty=nameWithType> or <xref:System.Windows.Forms.FlatStyle.Popup?displayProperty=nameWithType> now use OS-defined colors in High Contrast theme when selected. Previously, text and background colors were not contrasting and were hard to read.
+- Controls contained within a `System.Windows.Forms.GroupBox` which has its `P:System.Windows.Forms.Enabled` property set to _false_ will now use OS-defined colors in High Contrast theme.
+- `T:System.Windows.Forms.ToolStripButton`, `T:System.Windows.Forms.ToolStripComboBox`, and `T:System.Windows.Forms.ToolStripDropDownButton` controls have an increased luminosity contrast ratio in High Contrast Mode. 
+- `T:System.Windows.Forms.DataGridViewLinkColumn` will by default use OS-defined colors in High Contrast mode for the `P:System.Windows.Forms.DataGridViewLinkColumn.LinkColor` property.
+
+NOTE: Windows 10 has changed values for some high contrast system colors. Windows Forms Framework is based on the Win32 framework. For the best experience, run on the latest version of Windows and opt in to the latest OS changes by adding an app.manifest file in a test application and uncommenting the following code:
+```
+    <!-- Windows 10 -->
+    <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
+```
+
+__Improved Narrator support__
+- Narrator will now announce the value of the `P:System.Windows.Forms.Keys.ShortcutKeys` property when announcing the text of a `T:System.Windows.Forms.ToolStripMenuItem`. 
+- Narrator will now indicate when a `T:System.Windows.Forms.ToolStripMenuItem` has its `P:System.Windows.Forms.ToolStripMenuItem.Enabled` property to _false_.
+- Narrator will now give feedback on the state of a check box when the `P:System.Windows.Forms.ListView.CheckBoxes` property is set to _true_.
+- Narrator Scan Mode focus order is now consistent with the visual order of the controls on the ClickOnce download dialog window.
+
+__Improved DataGridView Accessibility support__
+- Rows in a `T:Systm.Windows.Forms.DataGridView` can now be sorted using the keyboard. Now a user can use the F3 key in order to sort by the current column.
+- When the `P:System.Windows.Forms.DataGridView.SelectionMode` property of a `T:System.Windows.Forms.DataGridView` is set to <xref:System.Windows.Forms.DataGridViewSelectionMode.FullRowSelect?displayProperty=nameWithType> the column header will change color to indicate the current column as the user tabs through the cells in the current row.

```

Comment:
The two links can be combined into one: \<xref:System.Windows.Forms.DataGridView.SelectionMode?displayProperty=nameWithType>
comma before "the column header"
will change --> changes